packages/simcore-sdk/src/simcore_sdk/node_ports_v2/links.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import Union
+
+from models_library.projects_nodes_io import UUID_REGEX, BaseFileLink, DownloadLink
+from models_library.projects_nodes_io import PortLink as BasePortLink
```

Comment:
I would advice to call PortLink in models_library with BASE prefix so that we are aware that any change there might have an impact in derived classes. Otherwise it is easy to assume that is an end class without derived